src/utils/idGenerator.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const idGenerator = (arr) => {
+  if (!arr.length) {
+    return 1;
+  }
+
+  const maxId = Math.max(...arr.map(({ id }) => id));
+
+  return maxId + 1;
+};
```

Comment:
You can use [uuid](https://www.npmjs.com/package/uuid)

Reply:
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/139622699/3f42fc5b-bedb-4992-99de-43741e0a3929)
id must be a number